### DISCUSSION TITLE
Secrets being moved to plugin in 2.11

### DISCUSSION THREAD

**Discussion by DanSheps** [netbox-community/netbox#5951] [General]
Hello all,

Just to give you a update, we are looking to move secrets into a plugin in 2.11.

Most of the work is completed on this as far as removing it from NetBox core.  You can review #5278 for the discussion surrounding it.

Feel free to discuss in here.

**Comment by jpobeda** [comment]
Hey @DanSheps, does this mean that by the time secrets get moved out of core there will be a plugin version ready?

**Reply by DanSheps** [reply]
Yes, I am currently working on bringing the functionality into a plugin.  I just need to decouple it from a large number of internal functions.

**Comment by jeremystretch** [comment]
We decided to push this back to v2.12, both to avoid introducing too much change at one time, and to allow more time for development of the replacement plugin.
